components/radio/themes/spectrum.css

```diff
@@ -13,15 +13,33 @@ governing permissions and limitations under the License.
 @container (--system: spectrum) {
   .spectrum-Radio {
     /* selection indicator */
-    --spectrum-radio-gray-default: var(--spectrum-gray-600);
+    /* --spectrum-radio-gray-default: var(--spectrum-gray-600);
     --spectrum-radio-gray-hover: var(--spectrum-gray-700);
     --spectrum-radio-gray-down: var(--spectrum-gray-800);
-    --spectrum-radio-gray-focus: var(--spectrum-gray-700);
+    --spectrum-radio-gray-focus: var(--spectrum-gray-700); */
 
     /* selected selection indicator */
-    --spectrum-radio-selected-gray-default: var(--spectrum-gray-700);
-    --spectrum-radio-selected-gray-hover: var(--spectrum-gray-800);
-    --spectrum-radio-selected-gray-down: var(--spectrum-gray-900);
-    --spectrum-radio-selected-gray-focus: var(--spectrum-gray-800);
+    /* --spectrum-radio-selected-gray-default: var(--spectrum-gray-700); */
+    /* --spectrum-radio-selected-gray-hover: var(--spectrum-gray-800); */
+    /* --spectrum-radio-selected-gray-down: var(--spectrum-gray-900); */
+    /* --spectrum-radio-selected-gray-focus: var(--spectrum-gray-800); */
+
+    --spectrum-radio-button-border-color-default: var(--spectrum-gray-600);
+    --spectrum-radio-button-border-color-hover: var(--spectrum-gray-700);
+    --spectrum-radio-button-border-color-down: var(--spectrum-gray-800);
+    --spectrum-radio-button-border-color-focus: var(--spectrum-gray-700);
```

Comment:
You already had similar properties defined above in lines 16-19, but I've renamed these custom properties to be a bit more explicit (and commented out the old ones so that you could look at them for reference).

The reasoning for this is because (in my opinion, at least!) we want our consuming applications to be able to look at a custom property and know exactly what it's used for. Hence, me explicitly adding `radio-button-border-color-*` here.

Reply:
I like this more clarified approach, thank you.